src/pages/Profile.js

```diff
@@ -1,22 +1,30 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React, {useContext} from 'react';
+import {Link} from 'react-router-dom';
+import {AuthContext} from "../context/AuthContext";
 
 function Profile() {
-  return (
-    <>
-      <h1>Profielpagina</h1>
-      <section>
-        <h2>Gegevens</h2>
-        <p><strong>Gebruikersnaam:</strong> hardcoded-test</p>
-        <p><strong>Email:</strong> hardcoded@test.com</p>
-      </section>
-      <section>
-        <h2>Strikt geheime profiel-content</h2>
-        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ab alias cum debitis dolor dolore fuga id molestias qui quo unde?</p>
-      </section>
-      <p>Terug naar de <Link to="/">Homepagina</Link></p>
-    </>
-  );
+
+    const {isAuth} = useContext(AuthContext);
+
+    return (
+        <>
+            {isAuth &&
+                <>
+                <h1>Profielpagina</h1>
+                <section>
+                <h2>Gegevens</h2>
+                <p><strong>Gebruikersnaam:</strong> hardcoded-test</p>
+                <p><strong>Email:</strong> hardcoded@test.com</p>
```

Comment:
Waarom render je pas alle content als isAuth true is? Het is niet fout, maar ik zou alleen het email adres conditioneel renderen. Ik zou het zo doen. 
```suggestion
                <p><strong>Email:</strong> {isAuth ? auth.user : "Loading" }</p>
```